pkg/cluster/cluster.go

```diff
@@ -443,6 +449,30 @@ func (os *ClusterSpec) Validate() error {
 	return nil
 }
 
+func (s *ClusterSpec) validateReplicationSlots() error {
+	if len(s.AdditionalReplicationSlotNames) == 0 {
+		return nil
+	}
+
+	rp := regexp.MustCompile("[_0-9a-z]+")
+
+	for _, replSlotName := range s.AdditionalReplicationSlotNames {
+		if trimmedSlotName := strings.TrimSpace(replSlotName); len(trimmedSlotName) == 0 ||
+			len(rp.ReplaceAllString(trimmedSlotName, "")) > 0 ||
```

Comment:
isn't `utils.IsValidReplSlotName(replSlotName)` enough? I don't see the need to trim space, \n etc... it should already be catched by it. If it's not enough then utils.IsValidReplSlotName should be fixed since the required check belong there.

I don't see the check for IsStolonName to error on repl slots starting with `_stolon` see my previous review comment: this function should just call two functions:  https://github.com/sorintlab/stolon/pull/434#discussion_r167164813

An unit test is needed to prove that all works correctly.